splunk/SplunkUtills/credentials.json

```diff
@@ -0,0 +1 @@
+{"host": "HOST", "port": PORT, "username": "USERNAME", "password": "PASSWORD"}
```

Comment:
You should state that the PORT that is needed here is the splunk management port and not web port.
This should be added in the notebook and module README.

Reply:
Done